<launch>
  <param name="use_sim_time" value="true"/>

  <group ns="object_detector">
    <rosparam file="$(find detect_and_track)/config/object_detection.yaml"/>
  </group>
  <node name="object_detector" pkg="detect_and_track" type="detect_node" output="screen">
  </node>
</launch>